src/lib.rs

```diff
@@ -462,6 +465,66 @@ impl Logger {
         self
     }
 
+    /// Sets the module path filter list. 
+    /// 
+    /// When any filter is matched as prefix of the log statement module path, the log
+    /// statement will be logged if log level allows.
+    /// Log statements not maching any filter will not be logged.
+    /// 
+    /// When not set (default) or set to empty Vec log statements will not be filtered 
+    /// by the module path.
+    /// 
+    /// # Example
+    /// Log only messages comming from this program.
+    /// 
+    /// ```rust
+    /// #[macro_use] extern crate log;
+    /// extern crate loggerv;
+    ///
+    /// fn main() {
+    ///     loggerv::Logger::new()
+    ///         .module_path_filters(vec![module_path!().to_owned()])
+    ///         .init()
+    ///         .unwrap();
+    ///
+    ///     error!("This is printed");
+    /// }
+    /// ```
+    pub fn module_path_filters(mut self, filters: Vec<String>) -> Self {
+        self.module_path_filters = filters;
+        self
+    }
+
+    /// Adds module path filter to the list of module path filters.
+    /// 
+    /// When any filter is matched as prefix of the log statement module path, the log
+    /// statement will be logged if log level allows.
+    /// Log statements not maching any filter will not be logged.
+    /// 
+    /// When not filters were added log statements will not be filtered 
+    /// by the module path.
+    /// 
+    /// # Example
+    /// Log only messages comming from this program.
```

Comment:
Oh, I would've thought the word `filter` implied filtering out what you passed in.
Maybe we could have an explicit `ignore` and `allow` prefixed name instead?

Reply:
i'll merge this but may tweak the naming.